src/V3Ast.h

```diff
@@ -2209,7 +2209,7 @@ AstNode* AstNode::addNext<AstNode, AstNode>(AstNode* nodep, AstNode* newp);
 AstNode* AstNode::addNext(AstNode* newp) { return addNext(this, newp); }
 void AstNode::addPrev(AstNode* newp) {
     replaceWith(newp);
-    newp->addNext(this);
+    newp->addNextHere(this);
 }
```

Comment:
I suspect addHereThisAsNext can be used instead, and this function deleted.

Even if not having a V3Ast edit function for only a couple usages feels wrong, all the edit ops should be commonly used.

Reply:
I removed it and replaced with addHereThisAsNext.